### ISSUE TITLE
seq2seq model generates always the same output

### ISSUE THREAD

**Issue by LucaSCostanzo** [totalgood/nlpia#39]
Hi,
I am pretty new to the world of NLP. I trained the encoder decoder model as described in chap 10 of the book, but the network always produces the same output, independently from the sentence in input. I tried with several combinations for the hyperparameters (learning rate, epochs, number of LSTM hidden neurons, batch, ecc), but nothing worked. Any suggestion?
